test/bats/tests/config/config_v1beta1_verifier_notation_akv.yaml

```diff
@@ -8,7 +8,7 @@ spec:
   parameters:
     verificationCertStores:
       certs:
-        - certstore-akv
+        - kmprovider-akv
```

Comment:
since this is a new CR, i m thinking if we can enforce "namespace must of provided". Since kmprovider is a namespace resource, we won't have to guess its namespace if its not specified.

Reply:
hmm.. the challenge is when we init the verifier, we don't know if its referencing a cerstore or a kpmprovider. Wondering if there is a way to enforce namespace

Reply:
Are we sure want to make the user always append namespace in front? I think the only scenario where a user would refere to a KMP outside the verifier's namespace is if they are referring to a clusterred KMP which would be introduced for multitenancy. I see that right now the verifier is cluster scoped. But I also see that there is previous functionality to get the cert store namespace and then append it if not present. https://github.com/deislabs/ratify/blob/8acc7f2091f8dd430d759a7e5505c54ded5b0e56/pkg/controllers/verifier_controller.go#L165

Reply:
The flow that I see already for certificate store is: 1. check if the resource has a namespace. 2. if not check if the ratify namespace env var is set. 3. when creating the notation verifier, append the namespace to the front of the certstore ref if one is not already provided.

For KMP, should we just follow the same behavior? I added a check in the notation trust store to check both the certificate store's certificate map as well as the KMP equivalent so there shouldn't be any ambiguity as to if its a KMP resource or a legacy Certificate Store.

Reply:
Akash and I had a offline discussion and agreed on KMP provider to mirror the CertStore behavior for this PR.  We should find another time to discussion the levels of namespace enforcement  as a part of Multi tenancy work.  

 our goal:
- intuitive and deterministic user experience.
- Straightforward application logic.

We want to review the valid combination of we should support and validate.  Cluster Verifiers (Cluster certStore, Namespaced certStore ) and Namespaced Verifiers (Cluster certStore, Namespaced certStore ). cc @binbin-li